charts/chirpstack/templates/postgres/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{- define "chirpstack.postgres.env" -}}
+- name: POSTGRES_PASSWORD
+  value: root
```

Comment:
Ehm, is this intended?

Reply:
Alright. Yes, you're right, it probably is confusing if we require two passwords.  
There is still the possibility to generate the password, but i guess using the postgres operator would make that obsolete.